### ISSUE TITLE
Feature Request: Regex-style table sidebar filtering

### ISSUE THREAD

**Issue by singjsong** [jakob/Postico#396]
e.g. `*_users`, `^user`, `users$`

Coming from Sequel Pro, to me, this is one of its most powerful features that I haven't seen in any Postgres GUIs.

**Comment by jakob** [comment]
`*_users` is not a standard regex. I assume you mean `.*_users`?

**Comment by singjsong** [comment]
Sorry, yeah, should have clarified; I was mixing standard glob syntax with regex. Something like `*_users` would return e.g. `new_users`/`old_users`/`young_users`, etc

**Comment by jakob** [comment]
No, Sequel Pro doesn't support glob syntax. They try matching a regex, and if it fails, they fall back to substring matching. See [here](https://github.com/sequelpro/sequelpro/blob/4a186517155223c48255e9952d002f465c567626/Source/SPTablesList.m#L2013)

Looks like a good idea...

**Comment by gajus** [comment]
This needs more visibility. Can this be added/ tracked in the `README.md`?

**Comment by jakob** [comment]
@gajus Sure. Just tell 12 more people to click the "thumbs up" button and it'll appear in the top ten when I update it the next time :)

**Comment by gajus** [comment]
There should be at least labels or some other means for people to find feature requests.

**Comment by gajus** [comment]
I don't understand how others work without this.

I have database with 150+ tables. When working on a particular problem, I want to be able to switch between related tables. Now I am using a new tab for every table. But that doesn't allow to visually navigate the tables quickly.

This is a quick feature to add.
